specs/protocol/configurability.md

```diff
@@ -41,23 +41,33 @@ The recommended way to deploy L1 contracts for an OP chain that meet the standar
+| [Max Game Clock Duration](https://specs.optimism.io/fault-proof/stage-one/fault-dispute-game.html#max_clock_duration)                                                       | The maximum amount of time that may accumulate on a dispute game team's chess clock.                                                                                                                                                                         | Static                              | 3.5 days                                                                                                                                                                                                      |                                                                                                                                                                                                                                                                                              |
+| [Game Clock Extension](https://specs.optimism.io/fault-proof/stage-one/fault-dispute-game.html#clock_extension)                                                             | The flat credit that is given to a dispute game team's clock if their clock has less than CLOCK_EXTENSION seconds remaining.                                                                                                                                 | Static                              | 3 hours                                                                                                                                                                                                       |                                                                                                                                                                                                                                                                                              |
+| Bond Withdrawal Delay                                                                                                                                                       | The length of time that must pass before dispute game bonds can be withdrawn.                                                                                                                                                                                | Static                              | 7 days                                                                                                                                                                                                        |                                                                                                                                                                                                                                                                                              |
```

Comment:
Does Bond Withdrawal Delay map to `DelayedWETH.DELAY_SECONDS`? If we can note in each entry to where in the code it's configured, that would be really helpful for chains (and us with OPSM) to make sure standard chain deployments are configured correctly. I'm indifferent to whether it's a link in the first column, or just in the notes column something like "Corresponds to `DelayedWETH.DELAY_SECONDS`"

Reply:
It maps to `faultGameWithdrawalDelay` in the deploy config JSON but that's not part of the specs and from there it does wind up  as `DelayedWETH.DELAY_SECONDS`.

I've added links to relevant sections of the specs - not sure how much that helps find the respective part of the implementation but should give a bit more detail.

Reply:
Thanks, we will take a pass at transcribing these to solidity in the OPSM code, and likely will ask @clabby or @Inphi to review afterwards